utils/config.ts

```diff
@@ -57,41 +57,52 @@ const NotificationHubPartitionsConfig = t.interface({
 export type BaseConfig = t.TypeOf<typeof BaseConfig>;
 const BaseConfig = t.intersection([
   t.interface({
-    APPINSIGHTS_INSTRUMENTATIONKEY: NonEmptyString,
-    // the internal function runtime has MaxTelemetryItem per second set to 20 by default
-    // @see https://github.com/Azure/azure-functions-host/blob/master/src/WebJobs.Script/Config/ApplicationInsightsLoggerOptionsSetup.cs#L29
-    APPINSIGHTS_SAMPLING_PERCENTAGE: withDefault(IntegerFromString, 5),
-
-    AzureWebJobsStorage: NonEmptyString,
-
-    NOTIFICATIONS_STORAGE_CONNECTION_STRING: NonEmptyString,
-    RETRY_ATTEMPT_NUMBER: IntegerFromString,
-
-    isProduction: t.boolean
+    NOTIFICATIONS_QUEUE_NAME: NonEmptyString,
+    NOTIFICATIONS_STORAGE_CONNECTION_STRING: NonEmptyString
   }),
-
   t.interface({
-    // a list of fiscal codes not to send notifications to
-    //   use case: when doing internal tests
-    FISCAL_CODE_NOTIFICATION_BLACKLIST: withDefault(
-      CommaSeparatedListOf(FiscalCode),
-      []
-    )
+    INTERNAL_STORAGE_CONNECTION_STRING: NonEmptyString,
```

Comment:
Do we need to set this variable in terraform [PR](https://github.com/pagopa/io-infrastructure-live-new/pull/715/files)?

Reply:
this var will be set by our terraform function plugin